README.md

```diff
@@ -1,2 +1,155 @@
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 OOP-School-Library <a name="about-project"></a>
+
+> This is an app built with ruby that allows you to Add new students or teachers, Add new books, Save records of who borrowed a given book and when
+returned
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Backend</summary>
+  <br />
+  <img src="https://img.shields.io/badge/Ruby-CC342D.svg?style=for-the-badge&logo=Ruby&logoColor=white" alt="ruby" />
+</details>
+
+### Key Features <a name="key-features"></a>
+
+<!-- > Describe between 1-3 key features of the application. -->
+
+- Add new students or teachers.
+- Add new books.
+- Save records of who borrowed a given book and when.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+<!-- > Describe how a new developer could make use of your project. -->
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+ git@github.com:dazehinn/OOP-school-library.git
+```
+
+### Install
+
+Install this project with:
+
+```
+  cd oop-school-library
+  bundle install
+```
+
+
+### Usage
+
+To run the project, execute the following command:
+
+```
+ruby my_list.rb
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```
+ ruby -I test
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+<!-- > Mention all of the collaborators of this project. -->
+
+👤 **Nshanji Hilary Ndzi**
+
+- GitHub: [dazehinn](https://github.com/dazehinn)
+- LinkedIn: [Nshanji Hilary Ndzi](https://www.linkedin.com/in/nshanji-hilary-ndzi/)
+- Twitter: [nketchogue](https://twitter.com/hilaryndzi)
+
+
+<!-- FUTURE FEATURES -->
+
+ ## 🔭 Future Features <a name="future-features"></a> 
+
+<!-- > Describe 1 - 3 features you will add to the project. -->
+
+- Preserve persons data.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/dazehinn/oop-school-library/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+ ## ⭐️ Show your support <a name="support"></a> 
+
+If you like this project, Please give it a ⭐️⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+ ## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Thanks [@microverse](https://www.microverse.org/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/dazehinn/oop-school-library/blob/Dev/LICENSE) licensed.
```

Comment:
- [ ] please, your MIT license is broken. Kindly fix it to keep your README file professional. 
![Screenshot (1124)](https://github.com/dazehinn/OOP-school-library/assets/65298048/2c73acb1-4cd4-49f7-a007-bc158d0810db)